modules/opensimplex/open_simplex_noise.h

```diff
@@ -75,7 +75,7 @@ class OpenSimplexNoise : public Resource {
 	void set_lacunarity(float p_lacunarity);
 	float get_lacunarity() const { return lacunarity; }
 
-	Ref<Image> get_image(int p_width, int p_height) const;
+	Ref<Image> get_image(int p_width, int p_height, Vector2 p_noise_offset) const;
```

Comment:
Nitpick but prefer to pass by reference rather than by value (see the same usage as in `get_noise_*` functions):

```suggestion
	Ref<Image> get_image(int p_width, int p_height, const Vector2 &p_noise_offset) const;
```

Reply:
Good feedback, thanks. I've checked in an update.